gears/data_utils.py

```diff
@@ -3,12 +3,83 @@
 import numpy as np
 import pandas as pd
 import scanpy as sc
+from scipy.sparse import csr_matrix
 
 from .utils import parse_any_pert
 
 sc.settings.verbosity = 0
 warnings.filterwarnings("ignore")
 
+def update_adata_for_gears(adata, control_name, cell_type):
+    """
+    Updates adata to required formatting for
+    GEARS PertData.new_data_process() method
+
+    Parameters
+    ----------
+    adata : AnnData
+        AnnData object containing raw data
+    control_name : str
+        Name of control perturbation in adata.obs.condition
+    cell_type : str
+        Name of cell type used in experiment
+    """
+
+    # Specify gene_name in var and cell_type in obs
+    adata.var['gene_name'] = adata.var.index.values
+    adata.obs['cell_type'] = cell_type
+
+    # Change name of control perturbation to 'ctrl'
+    adata.obs.condition = adata.obs.condition.astype(str)
+    for i in range(adata.n_obs):    
```

Comment:
Just a suggestion here, but maybe create a new column for this. This would help keeping track of which condition was set to control. Not super important though !

Reply:
Good idea!